test/case/ietf_routing/ospf_default_route_advertise/topology.dot

```diff
@@ -0,0 +1,33 @@
+graph "2x2" {
+	layout="neato";
+	overlap="false";
+	esep="+20";
+	size=10
+
+        node [shape=record, fontname="DejaVu Sans Mono, Book"];
+	edge [color="cornflowerblue", penwidth="2", fontname="DejaVu Serif, Book"];
+
+	PC [
+	    label="PC | { <mgmt1> mgmt1 | <data1> data1 | <data2> data2 |  <mgmt2> mgmt2 }",
+	    pos="20,80!",
+	    kind="controller",
+	];
+
+        R1 [
+	    label="{ <mgmt> mgmt | <data> data  | <link> link } | R1 \n 192.168.100.1/32 \n 10.10.10.10/32\n(lo)",
+	    pos="250,80!",
```

Comment:
Very minor thing here (if you want to). Set "y" coordinate on R1 to 85 to make the lines perfectly horizontal :)